guides/doc-Content_Management_Guide/topics/QuickStart.adoc

```diff
@@ -92,3 +92,104 @@ Default_Organization_Centos_Stream_BaseOS_x86_64_os               BaseOS x86_64
 Uploading Enabled Repositories Report
 ----
 . Check the `/etc/yum.repos.d/redhat.conf` and ensure that the appropriate repositories have been enabled.
+
+==== Using Content Views
+
+{Project} uses the concept of Content Views to create snapshots of a set of repositories and make them available to hosts registered to it.
+
+For example, if you want a group of hosts to have access to a group of repositories at some specific versions, add the needed repositories to a content view, publish and promote to a lifecycle environment.
+You can now register hosts to this content view and the hosts will have access to all the repositories at the locked version in the content view.
+
+This enables you to independently update repositories on the server while your hosts are still consuming the locked versions of the repositories in the published content views.
+
+If you do not use custom content views and lifecycle environments, all your hosts will always be pointed to the Default Content View and Library environment which is a snapshot of all the content at their latest version on the server.
+
+Please refer to the following chapters for more information on content management using content views and application life cycles.
+
+* xref:Managing_Content_Views[]
+* xref:Creating_an_Application_Life_Cycle[]
+
+==== Creating a Content View
+
+Use the following procedure to create a Content view.
+
+. In the {Project} web UI, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the *Name* field, enter *Centos content view*.
+. In the *Description* field, enter a description for the content view.
+. Click *Save*.
+
+==== Adding Repositories
+
+Use the following procedure to add the upstream Centos repositories to your Content view.
+
+. In the {Project} web UI, navigate to *Content* > *Content Views* > Select *CentOS content view* > *Yum Content* Dropdown> *Repositories*.
+. Click on *Add* section on the Repository selection page.
+. Locate the *AppStream x86_64 os* and *BaseOS x86_64 os* repositories.
+. Select both repositories
+. Click *Add Repositories*
+
+==== Publishing the Content View
+
+Use the following procedure to publish the Content view.
+
+. In the {Project} web UI, navigate to *Content* > *Content Views* > *Select the desired Content View*
+. Click on *Publish New Version* button on the top right of page.
+. Click *Save*
+
+This will publish Version 1 of the Content View and promote it to the Library Environment.
```

Comment:
```suggestion

```
We don't write endings like this unless it is pointing the user to where they can find it.